pkg/operator/revisioncontroller/revision_controller.go

```diff
@@ -89,12 +89,14 @@ func (c RevisionController) createRevisionIfNeeded(ctx context.Context, recorder
 
 	// check to make sure that the latestRevision has the exact content we expect.  No mutation here, so we start creating the next Revision only when it is required
 	if isLatestRevisionCurrent {
+		recorder.Eventf("LatestRevision", "returning early, %d triggered and up to date", latestAvailableRevision)
```

Comment:
I added this for debugging, but it's likely to be very noisy.  Perhaps just a log message at v(4) so it's not available by default?